ci/scripts/driver.sh

```diff
@@ -89,14 +89,14 @@ for pr in ${pr_list}; do
     if [[ "${driver_PID}" -ne 0 ]]; then
       echo "Driver PID: ${driver_PID} no longer running this build having it killed"
       if [[ "${driver_HOST}" == "${host_name}"  ]]; then
-        pstree -A -p "${driver_PID}" | grep -Eow "[0-9]+" | xargs kill || true
-        sleep 30
+        #shellcheck disable=SC2312,SC2312
```

Comment:
```suggestion
        # shellcheck disable=SC2312
```

Reply:
I was doubling that trying to get it to stop complaining still.